ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -137,20 +155,28 @@ private void EffectCombo_SelectionChanged(object sender, SelectionChangedEventAr
         {
             await ResetEffectsAsync();
 
-            string typeName = null;
+            VideoEffectDefinition videoEffectDefinition = null;
 
             switch (effect)
             {
-                case displacementEffect: typeName = typeof(DisplacementEffect).FullName; break;
-                case rotatingTilesEffect: typeName = typeof(RotatedTilesEffect).FullName; break;
+                case displacementEffect:
+                    videoEffectDefinition = new VideoEffectDefinition(typeof(DisplacementEffect).FullName, new PropertySet());
+                    break;
+                case rotatingTilesEffect:
+                    videoEffectDefinition = new VideoEffectDefinition(typeof(RotatedTilesEffect).FullName, new PropertySet());
+                    break;
+                case variableEffect:
+                    variableEffectValueSlider.Visibility = Visibility.Visible;
+                    effectPropertySet = new PropertySet();
```

Comment:
I will move the initialization of the propset to the loaded event and use the same one for all effect initialization. Since the other effects don't use any of the keys, there wont be any conflict of value type.